### ISSUE TITLE
"Invalid Twitter cookie!" error (possibly due to migration from twitter.com to x.com ?)

### ISSUE THREAD

**Issue by leomignot** [medialab/minet#973]
**TL;DR**
Getting "Invalid Twitter cookie!" error, likely due to the migration from twitter.com to x.com and changes in the cookie name/format. Setting the x.com cookie as an environment variable seems to fix the issue.

**Description**
When attempting to scrape tweets using minet, I recently encountered the following error message:
```
Invalid Twitter cookie!
Try giving another browser to --cookie and sure you are correctly logged in.
```
Despite trying to give a different browser to --cookie as suggested, the issue persists.

**Possible cause of error**
The problem likely stems from the recent (last week ?) transition of "twitter.com" to "x.com". This change has resulted in the format of their cookies being altered. It seems that minet is not correctly identifying or retrieving the necessary cookies due to the name and URL change.

**Workaround**
As a temporary solution, I was able to resolve the problem by manually setting the x.com cookie as an environment variable, which seems to confirm that this is the source of the problem (but I could be wrong).

I'm posting this issue in case others have the same problem and if you ever get the chance to fix it (even if it must get hard to fix things as fast as He-Who-Must-Not-Be-Named breaks them).

**Comment by Yomguithereal** [comment]
@leomignot I cannot reproduce because it depends on when and whence you logged into the website (domain AND geographic location, lol), but I suspect you are right.

I have pushed a commit just now. Would you be able to install it and test whether it fixes the issue? Tell me if you don't know how and I will release a version so you can test.

**Comment by leomignot** [comment]
@Yomguithereal : thanks for your quick reply. I'm not very good at this but I installed a dev environment following the instructions from your CONTRIBUTING.md.
I was able to give it a try (for example by running `lminet twitter scrape tweets “from:medialab_ScPo” --limit 10 > tweets.csv`) and it did the trick for me.

**Comment by Yomguithereal** [comment]
Good. I will close this issue then and release a v2.0.4.
